ods_ci/utils/scripts/ocm/ocm.py

```diff
@@ -1331,9 +1323,7 @@ def get_all_osd_versions(self):
             ).split("-")[0]
             if version:
                 osd_versions_dict[".".join(version.split(".")[:2])] = version
-                latest_osd_versions_data[
-                    str(self.osd_major_version)
-                ] = osd_versions_dict
+                latest_osd_versions_data[str(self.osd_major_version)] = osd_versions_dict
```

Comment:
```suggestion
                latest_osd_versions_data[str(self.osd_major_version)] = (
                    osd_versions_dict
                )
```

this is what the linter wants to see

Reply:
it's a stupid change; we should reconfigure black to not do things like ^^^; allow longer lines?

Reply:
we possibly can; not sure what is the default right now, my guess would be 80;  ... 120 characters seem like a normal length these days; not sure whether we want more...